README.md

```
@@ -84,8 +84,44 @@
+    - `file.path` - path to the file
+    - `file.created` - date when the file was created
+    - `file.accessed` - date when the file was last accessed
+    - `file.modified` - date when the file was last modified
+    - All other fields are from frontmatter data
+
+#### MD_LINKS
+
+- `FROM MD_LINKS("<path>")`
+- This will find all the links in markdown files in the specified `<path>`. Each link is a separate row.
+- FIELDS:
+    - `file.*` - file data same as above
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)